promql/parse.go

```diff
@@ -174,29 +177,39 @@ func (p *parser) error(err error) {
 	panic(perr)
 }
 
+func tokenDesc(i item) string {
+	ts := i.typ.String()
+	if strings.Contains(ts, "<") {
```

Comment:
Wouldn't this also match `itemLSS`? The implicit string encoding to signal tokens without default representation is making me a bit nervous...